lua/entities/entity_fishing_rod/cl_init.lua

```diff
@@ -74,7 +93,7 @@ function ENT:HUDPaint()
 	end
 
 	surface.SetFont("fixed_Height_Font")
-	local textBelow = "Total Catch: " .. ply.fishingmod.catches .. "\nMoney: " .. (math.Round(ply.fishingmod.money,2) or "0") .. "\nLevel: " .. ply.fishingmod.level .. "\nLength: " .. tostring(math.Round((self:GetLength() * 2.54) / 100 * 10) / 10) .. depth .. catch
+	local textBelow = "Total Catch: " .. ply.fishingmod.catches .. "\nMoney: " .. (math.Round(ply.fishingmod.money) or "0") .. "\nLevel: " .. ply.fishingmod.level .. "\nLength: " .. tostring(math.Round((self:GetLength() * 2.54) / 100 * 10) / 10) .. depth .. catch

```

Comment:
Why did you remove the amount of decimals shown?

Reply:
@Cynosphere 's request